src/services/ansiblePlaybook.ts

```diff
@@ -0,0 +1,179 @@
+import * as child_process from 'child_process';
+import { ExecException } from 'child_process';
+import { URL } from 'url';
+import { promisify } from 'util';
+import {
+  Connection,
+  Diagnostic,
+  DiagnosticSeverity,
+  Position,
+  Range,
+} from 'vscode-languageserver';
+import { TextDocument } from 'vscode-languageserver-textdocument';
+import { withInterpreter } from '../utils/misc';
+import { WorkspaceFolderContext } from './workspaceManager';
+const exec = promisify(child_process.exec);
+
+/**
+ * Acts as an interface to ansible-playbook command.
+ */
+export class AnsiblePlaybook {
```

Comment:
```suggestion
export class AnsiblePlaybookCmd {
```
(otherwise, why does a playbook have a connection?)

Reply:
Following the convention in the project, I decided to name it accordingly.